packages/runtime/src/extensibility/facades/transport/IdentityRecoveryKitsFacade.ts

```diff
@@ -0,0 +1,19 @@
+import { Result } from "@js-soft/ts-utils";
+import { Inject } from "@nmshd/typescript-ioc";
+import { TokenDTO } from "../../../types";
+import { CreateIdentityRecoveryKitRequest, CreateIdentityRecoveryKitUseCase, DoesIdentityRecoveryKitExistResponse, DoesIdentityRecoveryKitExistUseCase } from "../../../useCases";
+
+export class IdentityRecoveryKitsFacade {
+    public constructor(
+        @Inject private readonly createIdentityRecoveryKitUseCase: CreateIdentityRecoveryKitUseCase,
+        @Inject private readonly existsIdentityRecoveryKitUseCase: DoesIdentityRecoveryKitExistUseCase
```

Comment:
Missing change: `doesIdentityRecoveryKitExistUseCase` instead of `existsIdentityRecoveryKitUseCase`.